lib/logstash/inputs/kafka.rb

```diff
@@ -17,7 +17,7 @@
 # |0.9       |2.0.0 - 2.3.x   | 3.x.x |SSL |Works with the old Ruby Event API (`event['product']['price'] = 10`)  
 # |0.9       |2.4.x - 5.x.x   | 4.x.x |SSL |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
 # |0.10.0.x  |2.4.x - 5.x.x   | 5.x.x |SSL |Not compatible with the <= 0.9 broker
-# |0.10.1.x  |2.4.x - 5.x.x   | 6.x.x |SSL |Not compatible with <= 0.10.0.x broker
+# |0.10.1.x  |2.4.x - 5.x.x   | 6.x.x |SSL, Kerberos SASL |Not compatible with <= 0.10.0.x broker
```

Comment:
@suyograo I think we need another line here. How about something like this for the last three rows?

Kafka 0.10.0.x | Plugin v5.x.x | SSL, Kerberos SASL (v5.1.0+)
Kafka 0.10.1.0 | Plugin v6.0.0 - 6.2.1 | SSL, Kerberos SASL (v6.1.0+)
Kafka 0.10.1.1 | Plugin v6.2.2 - 6.x.x | SSL, Kerberos SASL